.github/workflows/main.yml

```diff
@@ -155,7 +155,8 @@ jobs:
 
       - name: Build documentation
         run: |
-          dub build -b ddox
+          mkdir build
```

Comment:
Ah I see the issue. You can just add a `build/.keep` file then, to make sure the directly is always present.